control/strategy/pool.go

```diff
@@ -227,6 +230,9 @@ func (p *pool) applyPluginMeta(a AvailablePlugin) error {
 	case plugin.StickyRouting:
 		p.RoutingAndCaching = NewSticky(cacheTTL)
 		p.concurrencyCount = 1
+	case plugin.ConfigRouting:
+		p.RoutingAndCaching = NewConfigBased(cacheTTL)
+		p.concurrencyCount = 3
```

Comment:
Good point. I agree it could be useful to provide concurrency count in global config. Default value could be 1. 

I think I found an error in ```pool.Eligible()``` that for ```p.concurrencyCount != 1``` function ```pool.Eligibility()``` does not work properly. 

Main condition:

```go
should := p.SubscriptionCount() / p.concurrencyCount
if should > p.Count() && should <= p.max {
    return true
}
return false
```
```p.SubscriptionCount()``` and  ```p.concurrencyCount``` are both ```int```

Let's say ```p.Count() = 1```, ```p.max = 3```, ```p.SubscriptionCount() = 4``` and  ```p.concurrencyCount = 3``` then ```should = 1```

Main condition is ```false```, thus ```pool.Eligible() = false``` while in my opinion it should be ```true```